Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -126,14 +126,7 @@ bool VFS::mount_root(FS& file_system)
     }
 
     m_root_inode = move(root_inode);
-    char device_name[32];
-    if (m_root_inode->fs().is_disk_backed()) {
-        auto& device = static_cast<DiskBackedFS&>(m_root_inode->fs()).device();
-        sprintf(device_name, "%d,%d", device.major(), device.minor());
-    } else {
-        sprintf(device_name, "not-a-disk");
-    }
-    klog() << "VFS: mounted root on " << m_root_inode->fs().class_name() << " (" << device_name << ")";
+    klog() << "VFS: mounted root on " << m_root_inode->fs().class_name() << " (" << static_cast<FileBackedFS&>(m_root_inode->fs()).file_description().absolute_path() << ")";
```

Comment:
The message should actually read "mounted root _from_ /dev/s.." — it's mounted from there onto `/`. Let's fix that too :slightly_smiling_face: